Wordpress/Helper/AttachmentHelper.php

```diff
@@ -78,4 +78,30 @@ public function getAttachmentUrl(Post $post, $size = 'post-thumbnail')
 
         return null;
     }
+
+    public function getAttachmentAltText(Post $post)
+    {
+        // Switch to correct blog
+        $originBlogId = $this->getManager()->getBlogId();
+        if ($originBlogId !== $post->getBlogId()) {
+            $this->managerRegistry->setCurrentBlogId($post->getBlogId());
+        }
+
+        $metadata = $this->getManager()->getRepository('KayueWordpressBundle:PostMeta')->findOneBy([
+            'post' => $post,
+            'key' => '_wp_attachment_image_alt',
+        ]);
```

Comment:
If I remember correctly `findOneBy` might throw `NoResultException` exception if no record found. 
A try catch block here would be nice.